src/data/array_type/circle_blur_paint_vertex.js

```diff
@@ -0,0 +1,51 @@
+// This file is generated. Edit build/generate-struct-arrays.js, then run `node build/generate-struct-arrays.js`.
+// @flow
+
+/* eslint-disable camelcase */
+
+const {Struct, StructArray} = require('../../util/struct_array');
+const StructArrayLayout = require('./struct_array_layout_4_1f');
+const {register} = require('../../util/web_worker_transfer');
+
+
+import type {SerializedStructArray} from '../../util/struct_array';
+
+class CircleBlurPaintVertexStruct extends Struct {
+    a_circle_blur: number;
+}
+
+(Object.defineProperty: any)(
+    CircleBlurPaintVertexStruct.prototype,
+    'a_circle_blur',
+    {
+        get: function () { return this._structArray.float32[this._pos4 + 0]; },
+        set: function (x) { this._structArray.float32[this._pos4 + 0] = x; }
+    }
+);
+
+class CircleBlurPaintVertexStructArray extends StructArrayLayout {
+    uint8: Uint8Array;
+    float32: Float32Array;
+
+    geta_circle_blur(index: number) {
+        return this.float32[index * 1 + 0];
+    }
+
+    static deserialize(input: SerializedStructArray): CircleBlurPaintVertexStructArray {
+        const structArray = Object.create(CircleBlurPaintVertexStructArray.prototype);
```

Comment:
This shouldn't really have to be duplicated each individual subclass, but it's currently necessary in order to create the deserialized object with the right prototype.  Maybe worth adding an `afterDeserialize` hook, which could be defined on the base class, so that the standard web worker transfer can take care of the `Object.create()` part.